src/js/EpubReader.js

```diff
@@ -1017,6 +1041,19 @@ BookmarkData){
         $('#zoom-custom a').on('click', enableCustom);
         $('.zoom-wrapper input').on('change', setCustom);
 
+        // UI tweaks for Hypothesis
+        window.hypothesisConfig = function () {
+            return {
+                onLayoutChange: function(state) {
+                    var $rightPageButton = $('#right-page-btn');
+                    $rightPageButton.css('right', state.width);
+                    if (!state.expanded) {
+                        $('nav').css('margin-right', state.width);
+                    }
+                }
+            };
+        };
+
```

Comment:
Obviously a bit "hacky". Another dirty implementation trick (albeit slightly less dirty, IMO) would be to place this code in the Hypothes.is plugin:
https://github.com/readium/readium-shared-js/blob/develop/plugins/hypothesis/main.js#L21
See merged PR:
https://github.com/readium/readium-shared-js/pull/396